app/routes/schema/stable/$.ts

```diff
@@ -0,0 +1,32 @@
+/*!
+ * Copyright © 2022 United States Government as represented by the Administrator
+ * of the National Aeronautics and Space Administration. No copyright is claimed
+ * in the United States under Title 17, U.S. Code. All Other Rights Reserved.
+ *
+ * SPDX-License-Identifier: NASA-1.3
+ */
+import { type DataFunctionArgs, redirect } from '@remix-run/node'
+
+import { getEnvOrDieInProduction } from '~/lib/env.server'
+
+type TagObject = {
+  name: string
+}
+
+export async function loader({ params: { '*': path } }: DataFunctionArgs) {
+  const GITHUB_API_TOKEN = getEnvOrDieInProduction('GITHUB_API_ACCESS')
+  const tags: TagObject[] = await (
+    await fetch('https://api.github.com/repos/nasa-gcn/gcn-schema/tags', {
+      headers: {
+        Accept: 'application/vnd.github+json',
+        Authorization: `Bearer ${GITHUB_API_TOKEN}`,
+      },
+    })
+  ).json()
```

Comment:
Are you sure that this request gives you the _most recent_ tag?

There's an API endpoint for the latest release: https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#get-the-latest-release

Also we already have a dependency on octokit, so you might as well use that rather than hitting the API endpoint directly.

Reply:
I was unaware of the octokit stuff, but that seems like its going to solve a lot of the issues I was having previously. I'll make use of it in the next commit. The tag api returns the values in descending order, so the first one in the list should be the most recent.

Are we going to be making releases off of tags?  If so I can use that api instead, and to be sure we are getting the most recent I can sort by the created date.

Reply:
Yes, we'll use GitHub releases.